doc/advanced/content/pcl_style_guide.rst

```diff
@@ -323,6 +323,40 @@ Command line usage:
    $ make format
 
 
+2.7. Includes
+^^^^^^^^^^^^^
+
+For consistent usage, headers should be included in the following order with alphabetical grouping ensured:
+
+1.  PCL headers
+
+    i.  All modular PCL includes, except main includes of common module
+    #.  The main PCL includes of common module
+2.  Major 3rd-Party components of tests and modules
+
+    i.  gtest
+    #.  boost
+    #.  Eigen
+    #.  flann
+3.  Major 3rd-Party components of apps
+
+    i.  Qt
+    #.  ui-files
+    #.  vtk
+4.  Minor 3rd-Party components
+
+    i.  librealsense
+    #.  ros/message_filters
+    #.  opencv/opencv2
+    #.  tide
+    #.  thrust
+    #.  OpenGL(UT)/GL(UT)
```

Comment:
I missed this (the user doesn't know you copy & pasted regular expression code ;-) ):
```suggestion
    #.  OpenGL, GL & GLUT
```

Reply:
I didn't copy. I just thought it was understood. :laughing: